### PR TITLE
Rename column support in Presto

### PR DESCRIPTION
**Pull request by nileema**

### CODE REVIEW

**Comment by nileema** [comment]
@electrum Updated

**Comment by electrum** [comment]
Looks good

File: presto-docs/src/main/sphinx/sql/alter-table.rst [hunk 1]
```diff
@@ -20,3 +21,7 @@ Examples
 Rename table ``users`` to ``people``::
 
     ALTER TABLE users RENAME TO people;
+
+Rename column ``id`` to ``user_id``::
+
+    ALTER TABLE users RENAME id TO user_id;
```
Comment (yan-stein) [inline presto-docs/src/main/sphinx/sql/alter-table.rst]: match "RENAME COLUMN" ?

**Comment by eteka75** [comment]
looks well

**Pull request closed without merging**